src/components/productoverview/ProductList.vue

```diff
@@ -35,17 +38,21 @@
         <!--{{> catalog/pop/filters-sidebar}}-->
       </div>
     </form>
-    <div v-if="isLoading">
+
+    <div v-else-if="isLoading"
+         class="loading-spinner">
       <img data-test="spinner" src="../../assets/img/spinner.gif"/>
     </div>
-    <div v-else-if="products && !products.results.length">
+
+    <div v-else-if="!isLoading && products && !products.results.length">
```

Comment:
Same as I mentioned in the comment that you resolved: if the previous conditional was `isLoading === true`, the next else-if can't be `isLoading === false`.

Reply:
And also the same as it was resolved: try to organize a bit the conditionals. The most efficient way would be:
if (isLoading)
else if (products && !products.results.length)
else